controllers/components/dashboard/dashboard_controller.go

```diff
@@ -108,7 +107,26 @@ func NewDashboardReconciler(ctx context.Context, mgr ctrl.Manager) error {
 	return nil
 }
 
-func CreateDashboardInstance(dsc *dscv1.DataScienceCluster) *componentsv1.Dashboard {
+func Init(platform cluster.Platform) error {
```

Comment:
Any reason to remove contextual logger as it is done now?

Reply:
mh, were we supposed to ignore the error and just log it ? right now it would cause the operator to fail to start

Reply:
> mh, were we supposed to ignore the error and just log it ? right now it would cause the operator to fail to start

I'm talking about fmt.Printf